storage/api.go

```diff
@@ -166,12 +173,21 @@ type StakingData struct {
 	Height int64
 	Epoch  beacon.EpochTime
 
+	Events           []*staking.Event
 	Transfers        []*staking.TransferEvent
 	Burns            []*staking.BurnEvent
 	Escrows          []*staking.EscrowEvent
 	AllowanceChanges []*staking.AllowanceChangeEvent
 }
 
+// RootHashData represents data for runtime processing at a given height.
+type RootHashData struct {
+	Height int64
+
+	Events []*roothash.Event
+	// TODO: Extend this interface to include runtime blocks.
```

Comment:
Maybe. What would we do with them? Just trying to understand the TODO.

Right now, it's a conscious decision to have each runtime have its own analyzer, so there's no reason for the consensus analyzer to fetch the runtime blocks. I think this design might be helpful once we start dealing with multiple versions of e.g. emerald, for which we'll want to run two slightly different emerald analyzers, each on a separate subset of heights. Meanwhile, the same consensus analyzer will process all of those blocks, assuming there's no breaking consensus version change. And vice versa, consensus might change and emerald might not.

Reply:
History was https://github.com/oasisprotocol/oasis-indexer/blob/699a599d14f8566cc27575a8c90f816721093e81/storage/api.go#L62-L64

Hmm, yeah, I think this can be removed now.

Reply:
Thanks for the explanation. Yes, sounds like we can remove this now.